port/linux/omrosdump_helpers.c

```diff
@@ -206,30 +214,45 @@ renameDump(struct OMRPortLibrary *portLibrary, char *filename, pid_t pid, int si
 
 	if (0 != createCoreFileNameRC) {
 		/* we could not derive the name from core_pattern */
-		portLibrary->str_printf(portLibrary,
-								filename,
-								EsMaxPath,
-								"Unable to derive system core filename from pattern in %s: \"%s\". "
-								"%s"
-								" Look for core file generated by child process with pid = %i",
-								corePatternFileName,
-								corePatternFormat,
-								derivedAbsoluteCorePath,
-								pid);
+		portLibrary->str_printf(
+				portLibrary,
+				filename,
+				EsMaxPath,
+				"Unable to derive system core filename from pattern in %s: \"%s\". "
+				"%s"
+				" Look for core file generated by child process with pid = %i",
+				corePatternFileName,
+				corePatternFormat,
+				derivedAbsoluteCorePath,
+				pid);
 		return 1;
 	}
 
 	/* wait until the core file is created, or timeout */
 	waitCoreRC = waitCore(derivedAbsoluteCorePath);
 	if (0 != waitCoreRC) {
 		/* the core file does not exist, bail */
-		portLibrary->str_printf(portLibrary,
-								filename,
-								EsMaxPath,
-								"The core file created by child process with pid = %i was not found. Expected to find core file with name \"%s\"",
-								pid,
-								derivedAbsoluteCorePath);
-		return 1;
+		if (corePatternPiped) {
+			/* Nothing inherently wrong with using a core pipe program, so direct the user to its man page. */
+			filename[0] = '\0';
+			portLibrary->nls_printf(
+					portLibrary,
+					J9NLS_INFO | J9NLS_STDERR,
+					J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND,
+					pid,
+					corePatternFileName,
+					corePatternFormatOriginal);
+			return 0;
+		} else {
+			portLibrary->str_printf(
+					portLibrary,
+					filename,
+					EsMaxPath,
+					"The core file created by child process with pid = %i was not found.  Expected to find core file with name \"%s\"",
+					pid,
+					derivedAbsoluteCorePath);
+			return 1;
+		}
 	}
 
 	/* at this point we know the file in derivedCorePath exists */
```

Comment:
While you're here, can you please correct this comment? I believe it should refer instead to `derivedAbsoluteCorePath`.